person.rb

```diff
@@ -7,9 +7,14 @@ def initialize(age, name)
     super()
     @age = age
     @name = name
+    @rental = []
```

Comment:
- [ ] It will be better if you name this array "rentals" instead of "rental". As I mentioned, a person can have multiple rentals.

Reply:
- [ ] Nice job @Owusu-Desmond , just as the previous reviewer mentioned kindly change `rental` to `rentals` remember in your rentals.rb, you are pushing to `person.rentals` not `person.rental`